spec/operators/catch-spec.ts

```diff
@@ -242,4 +242,15 @@ describe('Observable.prototype.catch', () => {
       done();
     });
   });
+
+  it('should allow types with operators such as mergeMap to flow correctly', () => {
+    type(() => {
+      Observable.from([{ a: 1 }, { a: 2 }, { a: 3} ])
+          .mergeMap(value => {
+              return Observable.of([value]);
+          })
+          .mergeMap(x => x)
+          .subscribe(x => x.a);
+    });
+  });
```

Comment:
I honestly don't understand why moving the one line fixes things.  It affects _all_ operators, not just catch. In fact the code I was working with at the time wasn't ever using catch.

I can drop the test itself, but thought it would better to have a test in the event something like it comes along again in the future.  Perhaps there's a better spec location to put it.